README.md

```diff
@@ -11,3 +11,38 @@ TODO: Add detailed app description and demo
 ![architecture diagram](images/architecture-diagram.png)
 [Image Link](https://excalidraw.com/#json=1c_DrTFZCSGprCvJNBYHn,B0sXrISCY8YdKgiTqy9xDA)
 _revised 9/4/2024_
+
+The structure of the event database is as follows:
+ ```json
+ {
+  "Event": {
+    "Name": "New Years Eve Party",
+    "Photo": "https://stackoverflow.com/questions/46585330/firestore-security-rules-for-public-and-private-fields",
+    "Description": "Get Ready for NYE 2025, dress classy and dance all night",
+    "Category": "Party",
+    "Start": {
+      "Date": "31.12.2024",
+      "Time": "23:00"
+    },
+    "End": {
+      "Date": "01.01.2025",
+      "Time": "06:00"
+    },
+    "Location": "Starling Hotel",
```

Comment:
Will location only be a string? Or will it also be an address, or longitude and latitude?

Reply:
Changed to name, lat and lon